applications/matter_bridge/src/bridged_devices_creator.cpp

```diff
@@ -0,0 +1,384 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
```

Comment:
I wonder if it will be better to split this creator for BLE and simulated devices. What do you think? It will be better manageable then. We can avoid using preprocessors in many places this way.

Reply:
I believe there are pros and cons of this solution, but yeah maybe it would be something nice to have. I created a follow-up to consider it: KRKNWK-17675, as I really want to merge this PR soon.